pages/ksphere/konvoy/latest/gpu/index.md

```diff
@@ -82,8 +75,7 @@ spec:
   nodePools:
   - name: gpu-worker
     gpu:
-      nvidia:
-        enabled: true
+      nvidia: {}
```

Comment:
```suggestion
      nvidia: {}
```
Does this need to have the `enabled: true` value pair?

Reply:
No this was on purpose, `enabled: true` was no longer correct.